.docker/entrypoint/lib/docker_image.rb

```diff
@@ -23,6 +20,14 @@ def initialize(name, tag, version)
         @fqn = "#{name}:#{tag}"
       end
 
+      def self.from_environment
+        docker_image_name = ENV.fetch('DOCKER_IMAGE_NAME')
+        docker_image_tag = ENV.fetch('DOCKER_IMAGE_TAG')
+        docker_image_version = ENV.fetch('DOCKER_IMAGE_VERSION')
```

Comment:
Can this use the same environment variable fetching as `context.rb`? Using the `EnvironmentVariables.new` and accessing properties?

Reply:
Yep, done in a7fbeadb320bbcc65461ecb0ef2d4d4a592929f0.